### DISCUSSION TITLE
SDR++ not starting

### DISCUSSION THREAD

**Discussion by fstieglitz** [AlexandreRouma/SDRPlusPlus#254] [Q&A]
Greetings.  I am probably doing something strange, but I thought I would ask anyway.
Just downloaded SDR++ Windows v1.0.3 Build 23.00.45 August 9, 2021.
I inserted a RTL SDR V3 dongle, and SDR++ correctly recognized it, and I selected it in SDR++.
Generic RTL2832U OEM (0).
Then, I clicked the "Play" button to start the radio.
Nothing seems to be happening.  No sounds, no waterfall.  Only the "Play" button switches into a "Stop" button, so I know it recognized my click.  Tuned to a local FM broadcast station, as a test.
What kinds of things should I look for?  Any suggestions or advice?
Many thanks,
Frank

**Comment by AlexandreRouma** [comment]
hello,
Did you install the proper rtl-sdr drivers using zadig? see the tutorial on rtlsdrblog.

**Reply by fstieglitz** [reply]
Ah ha!  I assumed since SDR++ recognized my RTL SDR dongle, that I did not require zadig.  I will try it out, and will report back here soon.  (Note: You see, I previously had a negative experience with zadig, as it seemed to permanently change some of my ports.  But... [insert gulp here]... I will try again.)
Thanks for the tip.

**Reply by fstieglitz** [reply]
Here is my follow-up.  I would like to suggest this is now resolved.
Yes.  Once I started zadig, SDR++ did start correctly!
As I hinted, I must have mis-read the instructions and thought SDR++ did not require zadig.  I will now continue poking around, and learn more about SDR++.  This sounds like a great project.
Thank you very much.
